cmd/occollector/app/collector/processors.go

```diff
@@ -63,7 +64,7 @@ func createExporters(v *viper.Viper, logger *zap.Logger) ([]func(), []exporter.T
 	return wrappedDoneFns, traceExporters, metricsExporters
 }
 
-func buildQueuedSpanProcessor(logger *zap.Logger, opts *builder.QueuedSpanProcessorCfg) (processor.SpanProcessor, error) {
+func buildQueuedSpanProcessor(logger *zap.Logger, opts *builder.QueuedSpanProcessorCfg) ([]func(), processor.SpanProcessor, error) {
```

Comment:
It would be useful if you could name this return parameters. With the addition `[]func()` I can't tell what this is and below in the good case you return `return nil, queuedSpanProcessor, nil` which is not intuitive.